src/v2/Apps/Consign/Routes/MarketingLanding/Components/PromoSpace.tsx

```diff
@@ -1,24 +1,22 @@
 import React from "react"
-import { Link, Text } from "@artsy/palette"
+import { Text, Box } from "@artsy/palette"
 import { SectionContainer } from "./SectionContainer"
-import { Media } from "v2/Utils/Responsive"
 
 export const PromoSpace: React.FC = () => {
   return (
-    <SectionContainer bg="black5" py={3}>
-      <Media greaterThanOrEqual="sm">
-        <Text variant="subtitle" color="black100" textAlign="center">
-          Gallerist or art dealer?{" "}
-          <Link href="https://partners.artsy.net">Become a partner</Link> to
-          access the largest global online art marketplace
+    <SectionContainer>
+      <Box borderBottom="1px solid" borderColor="black60" width="100%">
+        <Text
+          width="60%"
+          textAlign={"center"}
```

Comment:
Super minor and non-blocking - just a note that props for strings don't need to be wrapped in `{ ... }` brackets